packages/docusaurus/lib/commands/start.js

```diff
@@ -119,7 +123,17 @@ module.exports = async function start(siteDir, cliOptions = {}) {
     disableHostCheck: true,
     overlay: false,
     host,
-    contentBase: path.resolve(siteDir, 'static'),
+    // https://webpack.js.org/configuration/dev-server/#devserverbefore
+    // eslint-disable-next-line
+    before(app, server) {
+      app.use(baseUrl, express.static(path.resolve(siteDir, 'static')));
+      // TODO: add plugins beforeDevServer hook
```

Comment:
The plugin hooks is something like this in v1

https://github.com/facebook/Docusaurus/blob/94a4ef55c0bfed997b9c7c4d7e32f98695e3cc30/packages/docusaurus-1.x/lib/server/server.js#L347-L349

So for blog plugin, the equivalent is something like

```js
app.use(baseUrl, express.static(path.resolve(siteDir, 'blog', 'assets'));
```